_posts/minutes/2016-09-16-minutes.markdown

```diff
@@ -0,0 +1,93 @@
+#### BAKERSFIELD COLLEGE
+
+#### Computer Science Club
+
+#### 1801 Panorama Drive, Business Building, Room 5
+
+#### Bakersfield, California 93305
```

Comment:
Chad, next time you see me on campus, ask me. I know how to make that change, I totally forgot to show you all the times I've seen you on campus.